lib/controller/prefs.js

```diff
@@ -27,6 +27,8 @@ const addTo = (program) => {
     .option('--lang <lang>', 'Change language')
     .option('--maxsize <size>', 'Change logger file maxsize')
     .option('--blacklist <keyword>', 'Add keyword to logger blacklist')
+    .option('--timeouttype <type>', 'Timeout type ( session ) ')
```

Comment:
An option of the form `--timeout-type` would be preferred. Caporal does support such options, see [here](https://github.com/mattallty/Caporal.js/blob/master/examples/pizza/pizza.js#L37)

Reply:
Ok, I've fixed this too